data/json/npcs/classes.json

```diff
@@ -4,7 +4,7 @@
     "id": "NC_NONE",
     "name": "No class",
     "job_description": "I'm just wandering.",
-    "traits": [ { "group": "BG_survival_story_EVACUEE" }, { "group": "NPC_starting_traits" } ],
+    "traits": [ { "group": "BG_survival_story_EVACUEE" }, { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
     "skills": [
```

Comment:
You can add politically incorrect demographics to ``NPC_starting_traits`` instead of adding it to every class.
And dont forget mutant NPCs from the mod, they want new look too.